utils/build-script-impl

```diff
@@ -1315,10 +1315,10 @@ function set_swiftpm_bootstrap_command() {
     swiftpm_bootstrap_command+=(
         --swiftc="${SWIFTC_BIN}"
         --sbt="${LLBUILD_BIN}"
-        --build="${build_dir}")
-    if [[ ! "${SKIP_BUILD_FOUNDATION}" && ! "${SKIP_BUILD_XCTEST}" ]] ; then
+        --build="${build_dir}"
+        --foundation="${FOUNDATION_BUILD_DIR}/Foundation")
```

Comment:
Can you swap the order of these so the output build dir is last and the input dependency parameters are first?